tensorrt_llm/_torch/modules/fused_moe/nccl_ep_utils.py

```
@@ -0,0 +1,410 @@
+    return _nccl_ep_installed
+
+
+def _nccl_ep_supports_int32_topk_idx() -> bool:
+    """Return True when the loaded libnccl_ep supports int32 input topk_idx."""
+    try:
+        import nccl
+        from packaging.version import Version
+
+        nccl_ep_info = nccl.get_version().nccl_ep
+        nccl_ep_version = nccl_ep_info.version if nccl_ep_info is not None else None
+    except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)